scripts/affiliate_server.py

```diff
@@ -55,6 +56,14 @@
 
 API_MAX_ITEMS_PER_CALL = 10
 API_MAX_WAIT_SECONDS = 0.9
+AZ_OL_MAP = {
+    'cover': 'covers',
+    'title': 'title',
+    'authors': 'authors',
+    'publishers': 'publishers',
+    'publish_date': 'publish_date',
+    'number_of_pages': 'pagination',
```

Comment:
This one may be a @cdrini + @hornc question -- is it actually `number_of_pages` on both sides?

Reply:
@mekarpeles On the Open Library side, `number_of_pages` is supposed to be an integer and `pagination` is a string which can contain multiple sequences and Roman numerals etc. I'm as sure what the source data formats are.

Reply:
I believe we should use `number_of_pages` on both sides given https://openlibrary.org/books/OL25389615M.json (`pagination: "xviii, [19-591 p."`) and the fact that most new OL editions do not even have a `pagination` attribute.